### PR TITLE
data:ssvali

### PR DESCRIPTION
**Pull request by ssvali**
Creating eddiehub linkfree url

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2807"></a>

### CODE REVIEW

**Review by mocdaniel — APPROVED** [review]
Looks good, let's wait for a maintainer to merge this.

**Review by Njong392 — APPROVED** [review]
Looks great! Thanks for contributing :tada:

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**
